.goreleaser.yaml

```
@@ -1,22 +1,78 @@
 # .goreleaser.yaml
-builds:
-- main: ./cmd/
-  goos:
-    - linux
-    - darwin
-  goarch:
-    - amd64
-    - arm64
-  env:
-    - CGO_ENABLED=0
-  ldflags:
-    - -X github.com/0xPolygonHermez/zkevm-node.Version={{.Version}}
-    - -X github.com/0xPolygonHermez/zkevm-node.GitRev={{.Commit}} 
-    - -X github.com/0xPolygonHermez/zkevm-node.BuildDate={{.Date}}
-    - -X github.com/0xPolygonHermez/zkevm-node.GitBranch={{.Branch}}
+project_name: cdk-validium-node
+
 release:
-  # If set to auto, will mark the release as not ready for production
-  # in case there is an indicator for this in the tag e.g. v1.0.0-rc1
-  # If set to true, will mark the release as not ready for production.
-  # Default is false.
-  prerelease: true
+  disable: false
+  draft: true
+  prerelease: auto
+
+builds:
+  - main: ./cmd/
+    binary: zkevm-node
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -w
+      - -X github.com/0xPolygon/cdk-validium-node.Version={{ .Version }}
+      - -X github.com/0xPolygon/cdk-validium-node.GitRev={{ .Commit }}
+      - -X github.com/0xPolygon/cdk-validium-node.BuildDate={{ .Date }}
+      - -X github.com/0xPolygon/cdk-validium-node.GitBranch={{ .Branch }}
```

Nitpick, though not sure will it do the trick:

```suggestion
      - -X github.com/0xPolygon/{{ .ProjectName }}.Version={{ .Version }}
      - -X github.com/0xPolygon/{{ .ProjectName }}.GitRev={{ .Commit }}
      - -X github.com/0xPolygon/{{ .ProjectName }}.BuildDate={{ .Date }}
      - -X github.com/0xPolygon/{{ .ProjectName }}.GitBranch={{ .Branch }}
```